doc/nrf/app_dev/device_guides/nrf54l/kmu_basics.rst

```diff
@@ -1,50 +1,57 @@
 .. _ug_nrf54l_developing_basics_kmu:
 
-KMU Key provisioning – basics, good practices and limitations
-#############################################################
+Introducion to KMU key provisioning
+###################################
 
-. contents::
+.. contents::
    :local:
    :depth: 2
 
-nRF54l devices are equipped with KMU which supports storing keys secretly and securely.
-This is valuable even for public keys as KMU is able to push a key directly to Cracken RAM, and even in case a key is supposed to be transferred through addressable RAM it limits possibility for attacking at the key secret a loot.
-Therefore, whenever possible KMU usage is recommended for managing secrets.
+The nRF54L devices are equipped with a Key Management Unit (KMU) that facilitates secure and confidential storage of keys.
+This feature is crucial not only for private keys but also for public keys, as the :re:`KMU can directly transfer a key to the CRACEN RAM<ug_nrf54l_crypto_kmu_cracen_peripherals>`.
+Even when keys must pass through addressable RAM, the KMU significantly reduces the risk of key exposure.
+Therefore, you should use KMU for managing secrets whenere possible.
 
 Revocable keys
 **************
 
-keys can be invalidating for further use. RPOLICY for such key is ‘revoked’.
-For some technical solutions a few keys of the same kind might be provisioned – these are generations (0, 1, 2 …) of the same key.
-If a given moment only one generation should be used. If given key generation is compromised – it shall be removed from being used by revocation.
-It is technical detail of given technical solution on how revocation scheme works, however it is recommended to provision one of generation as not revocable (‘locked’)
-to make impossible DoS on availability of the key for given technical solution (if desirable).
+Keys can be set as invalid for further use with a revocation policy (RPOLICY), marked as ``revoked``.
```

Comment:
if the key has RPOLICY equal ``revoked`` then it can be invalidated (in run-time)